server/services/backlinks.js

```diff
@@ -0,0 +1,97 @@
+// @flow
+import { difference } from 'lodash';
+import type { DocumentEvent } from '../events';
+import { Document, Revision, Backlink } from '../models';
+import parseDocumentIds from '../../shared/utils/parseDocumentIds';
+
+export default class Backlinks {
```

Comment:
🍖 New backlinks service processes document events and creates / removes backlinks as needed